documentation/source/users/arkane/input.rst

```diff
@@ -119,12 +119,16 @@ Model Chemistry                                  AEC   BC   SOC  Freq Scale Supp
 ``'B3LYP/6-31G(d,p)'``                            v    v         v (0.961)  H, C, O, S
 ``'MRCI+Davidson/aug-cc-pV(T+d)Z'``               v         v               H, C, N, O, S
 ``'wb97x-d/aug-cc-pvtz'``                         v         v               H, C, N, O
+``DLPNO-CCSD(T)/def2-tzvp``                       v    v    v               H, C, N, O, S, F, Cl
```

Comment:
Do you mind placing DLPNO after `B-CCSD(T)-F12/aug-cc-pVnZ` so all CCSD and all wb97 methods are adjacent?